### PR TITLE
feat: initial redesign

### PR DESCRIPTION
**Pull request by Polostor**
# Short description
Initial redesign phase where we (@polostor & [emma](https://www.behance.net/emmzavo)) reworked the page design to have a bit of graphical design.

# Changes
feat: add font Share Tech Mono
feat: add colour palette 
feat: add artwork as featured image
feat: add responsive design features
feat: use LFS for binaries

# Static visuals
Google fonts - [Share Tech Mono](https://fonts.google.com/specimen/Share+Tech+Mono?query=share+)
Emma's design

Duplicate of #10

### CODE REVIEW

File: static/header_short.png
Comment (Indy2222) [inline static/header_short.png:1]: Looks like these are still not tracked by LFS.

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,3 @@
+static/header_short.png filter=lfs diff=lfs merge=lfs -text
```
Comment (Indy2222) [inline .gitattributes]: Please track all PNG files with `static/*.png`

**Review by Indy2222 — CHANGES_REQUESTED** [review]

File: static/header_short.png
Comment (Polostor) [reply static/header_short.png:1]: > peta@Petas-MacBook-Pro DE-web % git lfs ls-files --all
> 911a85f56c * static/ShareTechMono-Regular.ttf
> 8be7b3e947 * static/header_short.png
> e6d4997a6d * static/header_wide.png

File: .gitattributes (hunk 1 above)
Comment (Polostor) [reply .gitattributes]: I added new image file type (`svg`) instead of `png`s

**Comment by Polostor** [comment]
New image is:
new image

File: templates/index.html
Comment (Indy2222) [inline templates/index.html:1]: Why do you change file permissions?

File: static/header_short.png
Comment (Indy2222) [reply static/header_short.png:1]: You are right, I got confused with the new GitHub UI which AFAIK used to display LFS tracked files differently.

**Comment by Indy2222** [comment]
In Firefox, I have probably unintended vertical lines next to the background image:

image

File: templates/index.html
Comment (Polostor) [reply templates/index.html:1]: the what? I have not changed permissions (explicitly)

**Review by Indy2222 — APPROVED** [review]

**Pull request merged**